meson/pkg.yaml

```diff
@@ -0,0 +1,25 @@
+name: meson
+dependencies:
+  - stage: base
+  - stage: libffi
+  - stage: python3
+steps:
+  - sources:
+      - url: https://github.com/mesonbuild/meson/releases/download/0.61.1/meson-0.61.1.tar.gz
```

Comment:
By a strange coincidence, Meson 0.61.2 was released about 20 minutes before this PR was opened.

Reply:
thanks for noticing that!